powersimdata/scenario/create.py

```diff
@@ -267,56 +249,54 @@ def set_builder(self, interconnect):
             if len(possible) != 0:
                 print("%s: %s" % (p, " | ".join(possible)))
 
-        self._scenario_info["interconnect"] = self.builder.name
+        self._scenario_info["grid_model"] = self.builder.grid_model
+        self._scenario_info["interconnect"] = self.builder.interconnect
 
 
 class _Builder(object):
     """Scenario Builder.
 
+    :param
```

Comment:
Extra `:param`?

Reply:
I forgot to document the grid model parameter

Reply:
Done